lib/administrate/order.rb

```diff
@@ -6,11 +6,13 @@ def initialize(attribute = nil, direction = nil)
     end
 
     def apply(relation)
-      if relation.columns_hash.keys.include?(attribute.to_s)
-        relation.order("#{attribute} #{direction}")
-      else
-        relation
-      end
+      return order_by_association(relation) if
+        !reflect_association(relation).nil?
```

Comment:
Are we able to use a positive check here, rather than checking for a non-negative?

e.g.: `reflect_association(relation).exists?`